src/views/SignMessageLedger.vue

```diff
@@ -0,0 +1,144 @@
+<template>
+    <div class="container">
+        <SmallPage id="sign-message">
+            <PageHeader>
+                {{ $t('Sign Message') }}
+            </PageHeader>
+
+            <PageBody>
+                <textarea id="message" readonly="readonly" :value="request.message"></textarea>
```

Comment:
Would be theoretically better to use a `class` instead of an `id` here, as if there were to be a second `SignMessageLedger` instance on the page, it would be a duplicate identifier. Not that it's an actual issue here though, as there won't be two `SignMessageLedger` instances at the same time :)